### PR TITLE
Sammelband Manifests

### PR DESCRIPTION
**Pull request by tpendragon**
Implements manifests for resources which have both canvases (IE cover pages) AND child scanned resources. This allows MVWs to be Sammelbands. Whether we want to make a Sammelband model is now up to if we need a new workflow or not.

### CODE REVIEW

File: app/presenters/multi_volume_work_show_presenter.rb [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-    ::ScannedResourceShowPresenter
+    DynamicShowPresenter.new
   end
 
   def viewing_hint
     'multi-part'
   end
+
+  class DynamicShowPresenter
+    def new(*args)
```
Comment (escowles) [inline app/presenters/multi_volume_work_show_presenter.rb]: Is there any reason why the body of this method can't just go in `file_presenter_class` above?

Comment (tpendragon) [inline app/presenters/multi_volume_work_show_presenter.rb]: Yeah - the branch is based on the argument given to #new on initialization, which isn't available in `file_presenter_class`

Comment (escowles) [inline app/presenters/multi_volume_work_show_presenter.rb]: OK, that makes sense.  Then this looks ready to merge -- can you squash the commits?

**Comment by tpendragon** [comment]
@escowles Squashed the two range commits, but I like the separation of the range implementation and the canvas implementation.

**Comment by jpstroop** [comment]
It's not clear to me whether we need a new workflow or not. Could we walk through it tomorrow?

**Pull request merged**